<launch>
 <!-- RosAria Node-->
 <node name="RosAria" pkg="rosaria" type="RosAria">
   <!-- <param name="port" value="192.168.1.32:7171" /> -->
  </node>

 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="robot_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="0" type="int"/>
  <param name="scale_linear" value="2" type="double"/>
  <param name="scale_angular" value="2" type="double"/>

  <param name="joystickMovements" value="/RosAria/cmd_vel" type="string"/>

  <node pkg="OVbC" type="teleop_robot" name="teleop"/>
</launch>
